meta/demo_days.md

```diff
@@ -0,0 +1,91 @@
+# Demo Days at Artsy
+
+Demo Days are a way of showing off shipped projects to the rest of the team. It's a collection of 4-6 projects,
+explained by the people who contributed to them; product managers, designers and programmers.
+
+There are three separate roles involved in pulling a demo day off:
+
+* Slides / Project Coordinator
+* MC
+* Technical Support
+
+Most of the work is done before-hand in preparing the deck, and deciding what to show. This is the role of the
+slides/project coordinator. They need to:
+
+* Start the process by going to the [Sprint Kickoff][kickoff] slides
+* Make sure to find the sprint that’s just ending
+* Get all the tech leads names
+* Make a message to everyone asking for what they can do
+* Something like: “@x, @y, @z - you were all tech leads on this sprint, does your projecy you have anything good to
```

Comment:
"projecy"

I don't love the word "good" here, since teams have a tendency to downplay the importance of what they've shipped. I'd prefer if we said something like "the rest of the company uses demo days to see what the entire engineering team is up to, what did your team ship this sprint that's important to share?"